samples/net/mqtt/src/modules/sampler/Kconfig.sampler

```diff
@@ -0,0 +1,27 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menuconfig MQTT_SAMPLE_SAMPLER
+	bool "Sampler"
+	default y
+
+if MQTT_SAMPLE_SAMPLER
+
+config MQTT_SAMPLE_SAMPLER_THREAD_STACK_SIZE
+	int "Thread stack size"
```

Comment:
This will show up as `Thread stack size` in menuconfig, which is ambiguous. Perhaps `Sampler thread stack size` instead

Reply:
But its under the "sampler" submenu